cincinnati/src/plugins/internal/graph_builder/github_openshift_secondary_metadata_scraper/plugin.rs

```diff
@@ -88,7 +88,7 @@ pub struct GithubOpenshiftSecondaryMetadataScraperSettings {
     /// Vector of regular expressions used as a positive output filter.
     /// An empty vector is regarded as a configuration error.
     #[default(DEFAULT_OUTPUT_WHITELIST.iter().map(|s| (*s).to_string()).collect())]
-    output_whitelist: Vec<String>,
+    output_allowlist: Vec<String>,
```

Comment:
FYI this is a breaking change in the configuration format (as shown in the test change below).
I don't know whether anybody is relying on this at the moment though.